inductiva/fluids/scenarios/coastal_area/coastal_area.py

```diff
@@ -1,27 +1,24 @@
 """Coastal area scenario."""
 
 from functools import singledispatchmethod
-import math
 import os
+import math
 import random
 import shutil
-from typing import Literal, Optional, Sequence
-from uuid import UUID
+import typing
+import uuid
 
 import numpy as np
-import scipy
 
 from inductiva import tasks
-from inductiva.scenarios import Scenario
-from inductiva.simulation import Simulator
-from inductiva.fluids.simulators import SWASH
-from inductiva.utils.templates import (TEMPLATES_PATH,
-                                       replace_params_in_template)
-from inductiva.fluids.scenarios.coastal_area.output import CoastalAreaOutput
-
-from inductiva.generative import diamond_square
-
-SCENARIO_TEMPLATE_DIR = os.path.join(TEMPLATES_PATH, "coastal_area")
+from inductiva import scenarios
```

Comment:
Right now, I still need `from inductiva import fluids` due to circular imports. I will keep like this to avoid messing around to much, and afterwards we can fix this in a more general picture.